docusaurus/docs/running-tests.md

```diff
@@ -272,12 +272,13 @@ Note that tests run much slower with coverage so it is recommended to run it sep
 
 ### Configuration
 
-The default Jest coverage configuration can be overridden by adding any of the following supported keys to a Jest config in your package.json.
```

Comment:
We no longer _only_ allow coverage overrides, hence, I've also updated the wording here!

On a similar note, I've noticed that a few people would like an option to easily ignore certain files when running tests as mentioned in this [StackOverflow post](https://stackoverflow.com/questions/55632676/how-to-ignore-certain-files-when-running-npm-test-in-create-react-app/55634512#55634512). 

@iansu @amyrlam what do you think about including [testPathIgnorePatterns](https://jestjs.io/docs/en/configuration#testpathignorepatterns-arraystring) in the override options? Happy to raise a PR for it.